src/bootstrap/test.rs

```diff
@@ -1527,7 +1527,7 @@ impl Step for RustcGuide {
     fn run(self, builder: &Builder<'_>) {
         let src = builder.src.join("src/doc/rustc-guide");
         let mut rustbook_cmd = builder.tool_cmd(Tool::Rustbook);
-        try_run_quiet(builder, rustbook_cmd.arg("linkcheck").arg(&src));
+        try_run(builder, rustbook_cmd.arg("linkcheck").arg(&src));
```

Comment:
This change is based on [this thread](https://github.com/rust-lang/rust/pull/63089#discussion_r308018890): I think we actually want to see the output here.

The only other user of `try_run_quiet` uses logic of the kind 
```rust
    if builder.config.verbose_tests {
        try_run(builder, &mut cmd)
    } else {
        try_run_quiet(builder, &mut cmd)
    }
```